manifests/e/evernote/yinxiang/7.0.6.4762/evernote.yinxiang.installer.yaml

```
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=MDSU.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: evernote.yinxiang
+PackageVersion: 7.0.6.4762
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: /norestart
+FileExtensions:
+- enex
+Installers:
+- Architecture: x86
+  ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
+  InstallerUrl: https://cdn-dl.yinxiang.com/YXWin6/public/Evernote_7.0.6.4762.exe
+  InstallerSha256: E9270E462512D4D0536EEC9A84FC44767F0B481EBAEB72AB36D34CC0F613A2AF
+  AppsAndFeaturesEntries:
+  - Publisher: Beijing Yinxiang Biji Technologies Co., Ltd.
+    DisplayName: Yinxiang Biji v. 7.0.6
+    DisplayVersion: 7.0.6.4762
+    InstallerType: msi
+    ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
```

I think it also supports this?
```suggestion
  AppsAndFeaturesEntries:
  - Publisher: Beijing Yinxiang Biji Technologies Co., Ltd.
    DisplayName: Yinxiang Biji v. 7.0.6
    DisplayVersion: 7.0.6.4762
    InstallerType: msi
    ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
  - Publisher: 北京印象笔记科技有限公司
    DisplayName: 印象笔记 v. 7.0.6
    DisplayVersion: 7.0.6.4762
    InstallerType: msi
    ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
```

@denelon ?

---

I'll check with the team. I don't know if we have an ability to key on locale or if we can use both entries for a single neutral locale installer like this.